Gemfile

```diff
@@ -31,17 +31,11 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 # -----------------------------------------------------------------------------
-;
 
+source "https://rubygems.org"
 
-source "http://rubygems.org"
+gemspec
 
-gem 'rgeo-activerecord', github: 'rgeo/rgeo-activerecord', branch: '1.0'
-gem 'mysql2', '>= 0.2.13', '< 0.4.0'
-gem 'activerecord', '>= 4.0', '< 4.2'
-
-group :test do
-  gem 'rake', '>= 0.9.2'
-  gem 'rdoc', '>= 3.12'
-  gem 'pry'
-end
+# This dependency is here because gemspecs can't use git sources.
+# It overrides the less specific dependency in gemspec.
+gem 'rgeo-activerecord', git: 'https://github.com/rgeo/rgeo-activerecord.git', branch: '1.0'
```

Comment:
It seems we still need the constraint in the Gemfile. Without it, one test fails:

```
RGeo::ActiveRecord::Mysql2SpatialAdapter::Tests::TestBasic#test_create_geometry_with_index:
ArgumentError: struct size differs
```

Looks like your last contribution to that branch didn't make it in to the last rgeo-activerecord 1.* release (1.2.0).